### ISSUE TITLE
Confirm button indent at wrong position

### ISSUE THREAD

**Issue by Chissanu** [charmbracelet/huh#367]
**Describe the bug**
When creating a confirm button that has long description, the button would shift to right depends on the length of description

**To Reproduce**
1. Create a confirmation form
2. Create a very long desciption

**Expected behavior**
The confirm buttons are at the left side

**Screenshots**
![image-1.png](https://github.com/user-attachments/assets/f591edca-95fb-41f5-946f-5b2b0e455315)

**Desktop (please complete the following information):**
 - OS: [Windows]

**Comment by eduardoagarcia** [comment]
I noticed the same thing, and it looks like it's hard-coded to be centered: https://github.com/charmbracelet/huh/blob/main/field_confirm.go#L271

Seems like adding a property to override the alignment via a new function called something like `withAlginment()` would be fairly easy update to the package? 🙏 🙏

**Comment by Chissanu** [comment]
The work around I did was to put title and description for NewConfirm into NewNote and have NewConfirm to only store the values with no title and description.

**Comment by prithvijj** [comment]
I've created a PR for it here https://github.com/charmbracelet/huh/pull/425,
Happy to get any thoughts!
Shall post on the discord as well for reviews

**Comment by bashbunni** [comment]
Hey just a heads up - @prithvijj did some great work on #425 that was merged in #427 and fixes this issue. You can now specify a position for your confirm buttons using `WithButtonAlignment`. For example,
```go
NewConfirm().Title("Are you sure you want to commit those changes?").WithButtonAlignment(lipgloss.Left)
```
